### DISCUSSION TITLE
Option to disable letter caching

### DISCUSSION THREAD

**Discussion by Golbinex** [lollipopkit/flutter_server_box#430] [Ideas]
In the SSH terminal, letters are locally cached before being sent to the server via space key. This leads to undesired behavior when typing a password when executing sudo commands. There is no way to type a password without appended space.

I would like an option to disable letter caching, instead letters being directly sent to the server as they're typed.

**Comment by lollipopkit** [comment]
Thanks, #445

**Comment by lollipopkit** [comment]
Please test [v1008](https://cdn.lolli.tech/serverbox/pkg/)

**Comment by Golbinex** [comment]
Works fine. Issue resolved, thanks.
